content/docs/tasks/multicluster/split-horizon-eds/index.md

```diff
@@ -49,6 +49,14 @@ This will be used to access pilot on `cluster1` securely using the ingress gatew
 
 1. Use Helm to create the Istio deployment YAML for `cluster1`:
 
+    {{< warning >}}
+    Enabling additional components necessary for multicluster operation may increase the resource footprint
+    of Istio control plane beyond the capacity of a default Kubernetes cluster created by following
+    [Platform setup](/docs/setup/kubernetes/platform-setup/).
+    In case Istio components aren't getting scheduled due to insufficient CPU or memory consider
+    adding more nodes to your cluster or upgrade to instances of a larger size as necessary.
+    {{< /warning >}}
+
     {{< warning >}}
```

Comment:
While you're at it, please remove this other warning about helm dependencies, which is no longer needed.

Reply:
done